client/src/templates/Challenges/codeally/show.tsx

```diff
@@ -60,7 +79,7 @@ class ShowCodeAlly extends Component<ShowCodeAllyProps> {
           className='codeally-frame'
           // eslint-disable-next-line @typescript-eslint/restrict-template-expressions
           sandbox='allow-modals allow-forms allow-popups allow-scripts allow-same-origin'
-          src={`https://codeally.io/embed/?repoUrl=${url}`}
+          src={`https://codeally.io/embed/?repoUrl=${url}${envVariables}`}
```

Comment:
Sending something that should be private via query parameter is [a bit sketchy](https://owasp.org/www-community/vulnerabilities/Information_exposure_through_query_strings_in_url). Is there another way we can send the token to CodeAlly?

Reply:
You're probably right. There's not another way at the moment. Another plan we had at one point was to have users manually input their token within coderoad. It was decided to go this route. Unless there's some significant issue with it, I think this will have to work.